src/olympia/api/tests/test_authentication.py

```diff
@@ -207,29 +209,6 @@ def test_success(self):
         user, _ = self._authenticate(token)
         assert user == self.user
 
-    def test_refresh(self):
```

Comment:
I can't keep it, because... it would pass (return a 200). As long as an `orig_iat` is included in this token, the refresh would work. It doesn't matter here though, because clients do not generate this token. 

I kept the test for the apikey variant, to prove that even if clients generate a token with an orig_iat, if we ever expose refresh, by accident or voluntarily, it still would not allow those tokens to be refreshed.